src/server/system_services/account_server.js

```diff
@@ -1234,8 +1234,14 @@ async function update_user(req) {
     let iam_path = requested_account.iam_path;
     let user_name = account_util.get_iam_username(requested_account.name.unwrap());
     // Change to complete user name
-    const new_username = account_util.get_account_name_from_username(req.rpc_params.new_username, requesting_account._id.toString());
-    account_util._check_username_already_exists(action, new_username, req.rpc_params.new_username);
+    const is_username_update = req.rpc_params.new_username !== undefined &&
+        req.rpc_params.new_username !== req.rpc_params.username;
+    if (is_username_update) {
+        const email_new_username_wrapped = account_util.get_account_name_from_username(
```

Comment:
email_new_username_wrapped this name kind of confusing

Reply:
It is the email field of the new username suggested, and it is wrapped in a Sensitive string.
Do you have another variable name to suggest?